roles/confluent.common/tasks/main.yml

```diff
@@ -31,6 +31,24 @@
     state: absent
     regexp: '^Defaults\s+requiretty.*$'
 
+# Process Archive File deployments
+- name: Create Confluent Platform install directory
+  file:
+    path: "{{archive_destination_path}}"
+    state: directory
+    mode: 0755
+  when: installation_method == "archive"
+
+# If the target directory (i.e. creates) doesn't exist then download and expand the remote archive into target
+- name: Expand remote Confluent Platform archive
+  unarchive:
+    src: "{{ confluent_archive_file_path }}"
+    remote_src: true
```

Comment:
I think this should be toggleable with a variable. Some users might want to get the tar on the control node

Reply:
and probably default to false... most cases it will be thru the url, or then control node. I think your ansible-pull -ing the tar onto the hosts is the least likely actually

Reply:
Good call. However, I think it should default to `true` because URLs don't work with `remote_src: false`. I just tested this using the molecule test `archive-plain-rhel` and the error is the following (note: I changed the \n\t references to actual newlines and tabs for readability).

```
fatal: [kafka-broker1]: FAILED! => {"changed": false, "msg": "Could not find or access 'https://packages.confluent.io/archive/5.5/confluent-5.5.1-2.12.tar.gz'
Searched in:
    /mnt/c/development/github.com/GuyIEX/cp-ansible/roles/confluent.common/files/https://packages.confluent.io/archive/5.5/confluent-5.5.1-2.12.tar.gz
    /mnt/c/development/github.com/GuyIEX/cp-ansible/roles/confluent.common/https://packages.confluent.io/archive/5.5/confluent-5.5.1-2.12.tar.gz
    /mnt/c/development/github.com/GuyIEX/cp-ansible/roles/confluent.common/tasks/files/https://packages.confluent.io/archive/5.5/confluent-5.5.1-2.12.tar.gz
    /mnt/c/development/github.com/GuyIEX/cp-ansible/roles/confluent.common/tasks/https://packages.confluent.io/archive/5.5/confluent-5.5.1-2.12.tar.gz
    /mnt/c/development/github.com/GuyIEX/cp-ansible/files/https://packages.confluent.io/archive/5.5/confluent-5.5.1-2.12.tar.gz
    /mnt/c/development/github.com/GuyIEX/cp-ansible/https://packages.confluent.io/archive/5.5/confluent-5.5.1-2.12.tar.gz on the Ansible Controller.
If you are using a module and expect the file to exist on the remote, see the remote_src option"}
```

Reply:
sounds good default to true then. Thanks for the change, also nice documentation w it!